R/sim_gs_n.R

```diff
@@ -505,6 +511,19 @@ create_cut <- function(...) {
   }
 }
 
+# Create cut functions from the design object
+cut_from_design <- function(x) {
```

Comment:
Hi @yihui, a minor comment: can we have `@param` for the argument `x`? Besides, can you provide an example for `cut_from_design`?